config.example.json

```diff
@@ -14,5 +14,41 @@
         "580736157470556161"
     ],
     "welcomeChannelId": "335964434331860992",
-    "botowners": ["314046311035043840"]
+    "botowners": ["314046311035043840"],
+    "supportChannelId": "247865182477484032",
+    "supportEmbedDescription": [
+        "**Things we do not help with:**",
+        "> • If you are experiencing issues with third party templates or modules, please ask the template authors for support. We cannot assist you with them.",
+        "> • When you are using a pirated template or module. (this will get you banned)",
+        "> • If you are using a very old version of NamelessMC. Newer versions came out with a reason so don't expect support for older versions.",
```

Comment:
> Newer versions came out with a reason so don't expect support for older versions.
This sounds overly snarky and doesn't really convey any information.

"If you are using an old version of NamelessMC. It is difficult to provide support for old versions, as the issue may have been solved in a more recent update." maybe?